utils/bashrc.d/01-aws.bashrc

```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env bash
 
+set -eEuo pipefail
+
 complete -C '/usr/local/aws-cli/v2/current/dist/aws_completer' aws
+
+set +eEuo pipefail
```

Comment:
We are we setting and unsetting at the end of the file? How are these scripts being sourced? Normally this pattern shouldn't be needed

Reply:
I agree but I remembered it failed on a third party script, so I removed the explicit check here as it didn't exist before.
I think it's better than not knowing